.appveyor.yml

```diff
@@ -0,0 +1,23 @@
+version: 0-{branch}+{build}
+
+shallow_clone: true
+
+environment:
+  MIX_ENV: test
+
+install:
+  - ps: Start-FileDownload "https://github.com/elixir-lang/elixir/releases/download/v1.4.2/Precompiled.zip"
+  - 7z x -oelixir Precompiled.zip
```

Comment:
If I try to do `cinst elixir --version 1.4.2` and then `refreshenv` it takes too long when downloading the Erlang package dependency, that's why I'm doing this, please keep in mind that AppVeyor already includes Erlang. 

If you have any idea on how to improve this process please let me know.